.gitignore

```diff
@@ -1,63 +0,0 @@
-# Locked files
```

Comment:
looks like you removed our beloved gitignore my mistake, could you add it back ?